src/screens/regimen_setup/styles.ts

```diff
@@ -51,3 +55,92 @@ export const Welcome = styled.Text`
   color: ${({ theme }) => theme.colors.FONT_DARK_COLOR};
   text-transform: capitalize;
 `;
+
+export const RegimenQuestionBody = styled.View`
+  flex: 1;
+  justify-content: space-evenly;
+  align-items: center;
+`;
+
+export const QuestionContainer = styled.View`
+  height: ${deviceHeight * 0.18}px;
+  width: 240px;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+export const QuestionTitle = styled.Text`
+  font-size: ${({ theme }) => theme.fonts.LARGE_SIZE + 15}px;
+  font-family: ${({ theme }) => theme.fonts.JOST_BOOK};
+  text-transform: uppercase;
+`;
+
+export const QuestionRelevanceHeader = styled.Text`
+  font-size: ${({ theme }) => theme.fonts.MEDIUM_SIZE}px;
+  font-family: ${({ theme }) => theme.fonts.JOST_BOOK};
+  font-weight: ${({ theme }) => theme.fonts.FONT_WEIGHT_HEAVY};
+  text-transform: uppercase;
+  opacity: 0.3;
+`;
+
+export const QuestionRelevanceText = styled.Text`
+  font-size: ${({ theme }) => theme.fonts.LARGE_SIZE}px;
+  font-family: ${({ theme }) => theme.fonts.CORMORANT_REGULAR};
+  text-align: center;
+`;
+
+export const AnswersContainer = styled.View`
+  height: ${deviceHeight * 0.5}px;
+`;
+
+export const AnswerOption = styled.TouchableOpacity`
+  width: ${deviceWidth * 0.8}px;
+  margin-bottom: 8px;
+  height: 50px;
+  border-radius: 5px;
+  background-color: #f4f4f4;
```

Comment:
Please use colors from the theme. if you think a color you want to use is not there, define it yourself.This is to avoid hard coding colors all over the code base, ok

Reply:
have replaced hard-coded colors with colors from the theme.